cmd/boost/direct_deal.go

```diff
@@ -70,11 +70,162 @@ var directDealAllocate = &cli.Command{
 				"Default is 60 days.",
 			Value: verifreg13types.MaximumVerifiedAllocationExpiration,
 		},
+		&cli.StringFlag{
+			Name:     "piece-file",
+			Usage:    "file containing piece-info[s] to create the allocation. Each line in the file should be in the format 'pieceCid,pieceSize,miner,tmin,tmax,expiration'",
+			Required: true,
```

Comment:
`Required `shouble `true` ->`false` 
AND
https://github.com/filecoin-project/boost/blob/b36396531578b7c8446cef52009fd270823ccc14/cmd/boost/direct_deal.go#L35
https://github.com/filecoin-project/boost/blob/b36396531578b7c8446cef52009fd270823ccc14/cmd/boost/direct_deal.go#L41